src/qt/ProgressDialog.h

```diff
@@ -14,15 +14,14 @@ class ProgressDialog : public QDialog, private Ui::ProgressDialog
 	Q_OBJECT
 
 public:
-    ProgressDialog(QWidget * parent , const string title,bool showCancelButton) :
```

Comment:
Why remove title and cancel button?

Reply:
uhm, tbh i don't quite remember, i may have been confused by a warning or copile errors that i cannot even reproduce anymore, after reverting this.

Sorry for that.